lib/modules/transcripts/rest_api.php

```diff
@@ -51,6 +66,13 @@ public function update_voices($request)
         return rest_ensure_response(['status' => 'ok']);
     }
 
+    public function get_transcript($request) {
+        $episode_id = $request->get_param('id');
+        $transcript = Transcript::get_transcript($episode_id);
```

Comment:
@alexander-heimbuch Is it by intention that you return the raw transcript data here? It should be easier to work with data that has been passed through [`prepare_transcript`](https://github.com/podlove/podlove-publisher/blob/beta/lib/modules/transcripts/model/transcript.php#L67-L78) (there are two modes, `flat` and `grouped`, which should probably be exposed by the API)

Reply:
✅